tests/test_autodiffcc.py

```diff
@@ -4,6 +4,11 @@
 from autodiffcc import ADmath
 from autodiffcc import AD
 
+def test_wrong_shape():
+    try:
+        t1 = AD(val = np.array([2,3,1]), der = np.array([-1,3]))
+    except Exception
```

Comment:
```suggestion
    except: Exception
```

The build failed travis because of the missing colon